Code/cmake/Modules/FindCairo.cmake

```diff
@@ -53,6 +53,15 @@ FIND_LIBRARY(CAIRO_LIBRARIES
           ${PC_CAIRO_LIBRARY_DIRS}
 )
 
+IF (UNIX AND NOT APPLE)
```

Comment:
The changes to this file should not be part of this PR

Reply:
I thought I had done that, but obviously I hadn't - sorry.